### PR TITLE
fix login vertical centering and subsequent breaks

### PR DESCRIPTION
**Pull request by Nick-Yawn**
Co-authored-by: Sam Kramer <SamuelRKramer@gmail.com>
Co-authored-by: Jon Moore <jonevanmoore@yahoo.com>
Co-authored-by: Mason Kogami <maskogami@gmail.com>
Co-authored-by: Nick Yuan <NicholasNYuan@gmail.com>

### CODE REVIEW

**Review by jonevanmoore — APPROVED** [review]
Now we have a centered login page👍🏼

**Review by MasonKogami — APPROVED** [review]
Login page and subsequent breaks needed fixing and we got 'em!

**Review by samuelrkramer — APPROVED** [review]
Burying the lede here, which is that we're also moving a lot of general CSS into general files, instead of general stuff in specific files

File: public/stylesheets/layout.css [hunk 1]
```diff
@@ -0,0 +1,15 @@
+html {
+  height: 100%;
```
Comment (samuelrkramer) [inline public/stylesheets/layout.css:2]: Well dog my cats, I had no idea this is why none of my stuff was full height.

**Pull request merged**